tests/Unit/Domain/CoordinateMaps/Test_DiscreteRotation.cpp

```diff
@@ -108,10 +115,6 @@ void test_is_identity() {
   INFO("Is identity");
   check_if_map_is_identity(
       CoordinateMaps::DiscreteRotation<1>{OrientationMap<1>{}});
-  CHECK(not CoordinateMaps::DiscreteRotation<1>{
-      OrientationMap<1>{
-          std::array<Direction<1>, 1>{{Direction<1>::lower_xi()}}}}
-                .is_identity());
```

Comment:
For my understanding: why is this check removed?

Reply:
Accident. I originally was going to require 1d also have `J >0`, which would mean this check was not useful. I'll add it back!